lib/publiccloud/provider.pm

```diff
@@ -379,7 +379,12 @@ sub terraform_apply {
     assert_script_run($cmd, $terraform_timeout);
     my $ret = script_run('terraform apply -no-color -input=false myplan', $terraform_timeout);
     unless (defined $ret) {
-        type_string(qq(\c\\));        # Send QUIT signal
+        if (is_serial_terminal()) {
+            type_string(qq(\c\\));    # Send QUIT signal
+        }
+        else {
+            send_key('ctrl-\\');      # Send QUIT signal
+        }
```

Comment:
As those lines are twice the same they can be a subroutine. But it's a minor priority suggestion 👍

Reply:
a grep shows:
```
os-autoinst-distri-opensuse> grn '\\c\\\\'
lib/publiccloud/azure.pm:226:            type_string(qq(\c\\));
lib/publiccloud/provider.pm:382:        type_string(qq(\c\\));        # Send QUIT signal
lib/publiccloud/provider.pm:440:        type_string(qq(\c\\));        # Send QUIT signal
tests/kernel/virtio_console_long_output.pm:65:            type_string(qq(\c\\));              # Send QUIT signal
```
The last one seems to be for `virtio_console` so no change needed. But the one in `azure.pm` might hit your case as well, WDYT?

Reply:
Shouldn't hurt to patch also azure.pm, you are right. I just updated my commit and started a new run https://openqa.suse.de/tests/4772971#details